pass_import.py

```diff
@@ -735,9 +863,9 @@ def argumentsparse(argv):
   Import data from most of the password manager. Passwords
   are imported in the existing default password store, therefore
   the password store must have been initialised before with 'pass init'""",
-    usage="%(prog)s [-h] [-V] [[-p PATH] [-c] [-C] [-s] [-e] [-f] | -l] [manager] [file]",
-    formatter_class=argparse.RawDescriptionHelpFormatter,
-    epilog="More information may be found in the pass-import(1) man page.")
+                                     usage="%(prog)s [-h] [-V] [[-p PATH] [-c] [-C] [-s] [-e] [-b              KeePassFile password][-f] | -l] [manager] [file]",
```

Comment:
The `-b` option is not needed. If there are binary files, it is needed to import them.